framework/db/Connection.php

```diff
@@ -1098,4 +1101,32 @@ public function __clone()
             $this->pdo = null;
         }
     }
+
+    /**
+     * @var ConnectionLoggerInterface
+     * @since 2.0.13
+     */
+    protected $_logger;
```

Comment:
I think you meant private instead protected. And this should be before declaring any functions in this very class.